### ISSUE TITLE
Produce PDFs for issue 5

### ISSUE THREAD

**Issue by elotroalex** [archipelagosjournal/archipelagos#43]
The biggest issue for producing these PDFs will be the new audio.html element. You can find this one in the Edmond and in the Goffe piece. I built it using the model of the image.html. This one is tricky in that it only has a link and a name. The solution I'm thinking of is just to put it as a bullet point list with linked titles. This makes a lot more sense for the Edmond piece than the Goffe piece. The other option is to ignore them and just use the footnote. What do you think? 

In any case, the crazy goal this year is to have the issue out by next Friday. It is ready in theory, all we need is those PDFs. On my end all I need is to get DOIs for these ones. I'm gonna do that right now, and push to branch issue5 when I'm done. I figured though you might want a headstart on this audio issue.

**Comment by Denubis** [comment]
I can't do much work on this this week -- sorry. Work is crazy. I'll do what I can, but... no, not friday.

**Comment by elotroalex** [comment]
Sounds good. We can probably start with a generation of the PDFs for those without sound, just using the normal process. I think I made a copy of the docker container a million years ago. I'm gonna see if I can dig it out to pinch hit. I'll keep you posted.

**Comment by Denubis** [comment]
Oh, I can run a generation thingo -- I'm just not committing to new code.

**Comment by elotroalex** [comment]
sweet. let me turn off the PDF for the Edmond, and tweak the Goffe (only 1
sound file) so that we can have a PDF now. brb.

**Comment by elotroalex** [comment]
Ok, we should be good. I replaced the audio player with a link to the audio file on the Goffe, and set the PDF to false for the Edmond, since the whole point of that one is the sound files. He said he's fine with it.

**Comment by Denubis** [comment]
You haven't run any of the QA stuff for this issue, have you?

**Comment by Denubis** [comment]
Ok, I'll... spend this afternoon getting the stupid issue compiling...

**Comment by elotroalex** [comment]
QA stuff?

**Comment by Denubis** [comment]
quality assurance, like no smartquotes and making sure that 

```

{% include image.html 
   img="issue05/prater/fig4.jpg" 
   title="Sunshine Records 45rmp"
   caption="Sunshine Records 45rmp" %}

{% include image.html 
img="issue05/prater/fig5.jpg"
title=Images of 45rpm records sometimes used over the track of Stephen Cheng's 'Always Together' in YouTube uploads."
caption="Figures 4 and 5. Images of 45rpm records sometimes used over the track of Stephen Cheng's 'Always Together' in YouTube uploads." %}

{% include image.html
img="issue05/prater/fig6.jpg"
title="BMN Collection"
caption="Figure 6. Another album cover sometimes used over the track of Stephen Cheng's 'Always Together' in YouTube uploads." %}

``` 
doesn't require half an hour of me trying to figure out what you're trying to do?

**Comment by Denubis** [comment]
aaaaaaah. You've left smartquotes in one, and escaped normal double quotes in the other. Why do you hate me?

**Comment by elotroalex** [comment]
Yeah, I think I got all the stupid curly brackets off. There were a lot. Lol. Missed those ones!

**Comment by Denubis** [comment]
And like, half of these don't have titles...

**Comment by elotroalex** [comment]
I should write a script to detect all the possible permutations of these.

**Comment by Denubis** [comment]
I have, and it complains when it breaks...

**Comment by Denubis** [comment]
It compiled, pushing. You should probably do manual inspection of prater-elegy and ajg, since I had to do manual interventions in both of those. I'm going to go make coffee and figure out breakfast. Audio for... later.

**Comment by elotroalex** [comment]
Good times. I'm making dinner now. Will check after I've fed the troops. Enjoy coffee.

**Comment by elotroalex** [comment]
Just took a good look. The issue is looking good. I'll skip the link checker for now, since I'm sure we'll have to revisit for that audio part. I'm sending to the copyeditor for her to look at the PDFs. I had to re-insert the escaped quotation marks in the Prater. No clue what's happening there, since after your fix some had it some didn't, but one thing was for sure: the html was not compiling.

**Comment by Denubis** [comment]
Alternative build route is to use github actions? Been doing a lot of work in it recently.
